tests/unit/conditions/test_return_value.py

```diff
@@ -23,6 +23,34 @@
 from nucypher.policy.conditions.lingo import ReturnValueTest
 
 
+def test_return_value_key():
```

Comment:
If we are supporting context variables as keys, you could also add a test to `test_return_value_test_with_context_variable_cant_run_eval` to ensure that of a context variable for a key it fails if not converted before use.